api/src/main/scala/Geoprocessing.scala

```diff
@@ -83,6 +83,24 @@ trait Geoprocessing extends Utils {
     }
   }
 
+
+  /**
+    * For an InputData object, returns a sequence of maps of min, avg, and max
+    * values for each raster, in the order of the input rasters
+    *
+    * @param   input  The InputData
+    * @return         Seq of map of min, avg, and max values
+    */
+  def getRasterSummary(input: InputData): Future[ResultSummary] = {
+    val aoi = createAOIFromInput(input)
+    val futureLayers = cropRastersToAOI(input.rasters, input.zoom, aoi)
+    val opts = getRasterizerOptions(input.pixelIsArea)
+
+    futureLayers.map { layers =>
+      ResultSummary(rasterSummary(layers, aoi, opts))
```

Comment:
Since `opts` is only used once, consider dropping the assignment and just inlining the function call.

Reply:
I was following the established patten from here:

https://github.com/WikiWatershed/mmw-geoprocessing/blob/70eebe713e819cb14559470b94ffc5f96883ba1d/api/src/main/scala/Geoprocessing.scala#L22-L30

and I appreciate the readability. Furthermore, I'm sure the compiler is optimizing it under the hood, so it probably doesn't have a runtime penalty.